CMakeLists.txt

```diff
@@ -731,6 +731,7 @@ set(SOURCES
         table/table_factory.cc
         table/table_properties.cc
         table/two_level_iterator.cc
+        test_util/mock_time_env.cc
```

Comment:
Shouldn't this be in TEST_LIB_SOURCES and not LIB_SOURCES?

Reply:
I was just copying from sync_point.cc. I'll try it and see if I have difficulty working out the public as well as internal dependencies.

Reply:
Maybe `TESTUTIL_SOURCE` for cmake.